services/git-service-impl/src/test/java/io/fabric8/launcher/service/git/AbstractGitServiceTest.java

```diff
@@ -514,6 +516,36 @@ public void deleteHook() throws Exception {
                 .containsExactly(hook1);
     }
 
+
+    @Test
+    public void cloneRepository() throws Exception {
+        // Just run this if outside simulation mode
+        Assume.assumeFalse("This test should only run if HoverFly is not in simulation mode",
```

Comment:
I added this assumption because I couldn't make Hoverfly work with the native `git` command